jabgui/src/main/java/org/jabref/gui/maintable/SearchGoogleScholarAction.java

```diff
@@ -0,0 +1,51 @@
+package org.jabref.gui.maintable;
+
+import java.io.IOException;
+import java.util.List;
+
+import javafx.beans.binding.BooleanExpression;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.StateManager;
+import org.jabref.gui.actions.SimpleCommand;
+import org.jabref.gui.desktop.os.NativeDesktop;
+import org.jabref.gui.preferences.GuiPreferences;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.logic.util.ExternalLinkCreator;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+import static org.jabref.gui.actions.ActionHelper.isFieldSetForSelectedEntry;
+import static org.jabref.gui.actions.ActionHelper.needsEntriesSelected;
+
+public class SearchGoogleScholarAction extends SimpleCommand {
+    private final DialogService dialogService;
+    private final StateManager stateManager;
+    private final GuiPreferences preferences;
+    private final ExternalLinkCreator externalLinkCreator;
+
+    public SearchGoogleScholarAction(DialogService dialogService, StateManager stateManager, GuiPreferences preferences) {
+        this.dialogService = dialogService;
+        this.stateManager = stateManager;
+        this.preferences = preferences;
+
+        this.externalLinkCreator = new ExternalLinkCreator(preferences.getImporterPreferences());
+
+        BooleanExpression fieldIsSet = isFieldSetForSelectedEntry(StandardField.TITLE, stateManager);
+        this.executable.bind(needsEntriesSelected(1, stateManager).and(fieldIsSet));
+    }
+
+    @Override
+    public void execute() {
+        stateManager.getActiveDatabase().ifPresent(databaseContext -> {
+            final List<BibEntry> bibEntries = stateManager.getSelectedEntries();
+            externalLinkCreator.getGoogleScholarSearchURL(bibEntries.getFirst()).ifPresent(url -> {
+                try {
+                    NativeDesktop.openExternalViewer(databaseContext, preferences, url, StandardField.URL, dialogService, bibEntries.getFirst());
+                } catch (IOException ex) {
+                    dialogService.showErrorDialogAndWait(Localization.lang("Unable to open Google Scholar."), ex);
```

Comment:
Please use notification service - to avoid dialogs getting in the way.

Current thinking: Show a dialog only if the user has only one choice to continue - or if the user has to decide something.

It is more an information / error message, we notifiy. -- Otherwise, we would need to distinguish how long the action can take from start to error - and show a dialog or a notification.